packages/@angular/cli/lib/config/schema.json

```diff
@@ -192,6 +192,22 @@
             "description": "Name and corresponding file for environment config.",
             "type": "object",
             "additionalProperties": true
+          },
+          "budgets": {
```

Comment:
With this scheme, how do I specify the error/warning levels? How do I specify my budget for initial vs total?